Week-3/Exercise-3.3/output.txt

```diff
@@ -0,0 +1,29 @@
+function createIncrement(){
+    let count=0;
+    function increment(){
+        count++;
+    }
+        
+    let message=`Count is ${count}`;
+    function log(){
+        console.log(message);
+    }
+    return[increment,log];
+}
+
+const[increment,log] = createIncrement();
+
+increment();
+increment();
+increment();
+log();// What is logged?
+
+----------------------------- calling increment
+Output: count is 0
+
```

Comment:
Multiline comments should be enclosed in /** **/

Reply:
Hi Tanish, Added multiline comments please approve.